ddl/ddl_worker.go

```diff
@@ -509,12 +509,24 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 
 			var err error
 			t := newMetaWithQueueTp(txn, w.tp)
+			// only general ddls allowed to be executed when TiKV is disk full.
+			if w.tp == addIdxWorker {
```

Comment:
What if `change column type`?

Reply:
Because currently the `modify/change column` is still in the normal queue, if set it not allowed, then it will block other ddls in the normal queue.
And  `modify/change column` will be moved to the same queue with `create index`, also dealt with by the same ddl worker.